layers/+lang/ess/README.org

```diff
@@ -69,3 +39,133 @@ hook:
+| ~SPC m s w~ | set the working directory for the REPL                                     |
+| ~SPC m s p~ | load the specified library into the REPL                                   |
+| ~SPC m s P~ | install the specified library                                              |
+
+** Evaluation Commands
+Send code to inferior process with these commands:
+
+| Key Binding | Description                                                                      |
+|-------------+----------------------------------------------------------------------------------|
+| ~SPC m c c~ | send knitr/sweave chunk and keep buffer focused                                  |
+| ~SPC m c C~ | send knitr/sweave chunk and switch to REPL in insert mode                        |
+| ~SPC m c d~ | send knitr/sweave chunk and step to next chunk                                   |
+| ~SPC m c m~ | mark knitr/sweave chunk around point                                             |
+| ~SPC m c n~ | next knitr/sweave chunk                                                          |
+| ~SPC m c N~ | previous knitr/sweave chunk                                                      |
+| ~SPC m s b~ | send buffer and keep code buffer focused                                         |
+| ~SPC m s B~ | send buffer and switch to REPL in insert mode                                    |
+| ~SPC m s d~ | send region or line and step (debug)                                             |
+| ~SPC m s D~ | send function or paragraph and step (debug)                                      |
+| ~SPC m s l~ | send line and keep code buffer focused                                           |
+| ~SPC m s L~ | send line and switch to REPL in insert mode                                      |
+| ~SPC m s r~ | send region and keep code buffer focused                                         |
+| ~SPC m s R~ | send region and switch to REPL in insert mode                                    |
+| ~SPC m s t~ | send thing at point (function) and keep code buffer focused                      |
+| ~SPC m s T~ | send thing at point (function) and switch to REPL in insert mode                 |
+| ~SPC m s :~ | execute a command in the REPL, displaying the results in a separate buffer       |
+| ~SPC m v d~ | display a dired-like buffer browsing the objects in the inferior session buffer. |
+| ~CTRL+j~    | next item in REPL history                                                        |
+| ~CTRL+k~    | previous item in REPL history                                                    |
+
+** Developer
+Useful, IDE like functions.
+| Key Binding | Description                                                                                |
+|-------------+--------------------------------------------------------------------------------------------|
+| ~SPC m d l~ | ~(ess-developer-load-package)~ load a specified directory with ~devtools::load_all~        |
```

Comment:
Please, avoid function names here. People usually far more interested in what it does 😸